example/lib/main.dart

```diff
@@ -30,7 +30,7 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   final String _text =
-      'Lorem ipsum https://flutter.dev\nhttps://pub.dev dolor https://google.com sit amet';
+      'Lorem ipsum https://flutter.dev\nhttps://pub.dev dolor https://www.google.de/search?q=flutter+&oq=flutter sit amet';
```

Comment:
Let's use ".com" here instead of ".de"

Reply:
Yes .com is better :). I still kept the `/search?q=flutter...` part to show off the functionality in the example.